aws-site-1-app-vm/terragrunt.hcl

```diff
@@ -0,0 +1,26 @@
+include "root" {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "github.com/jeffgiroux/terraform-f5xc-aws-app-vm.git?ref=v0.0.1"
+}
+
+dependencies {
+  paths = ["../aws-base-1","../aws-vpc-site-1"]
```

Comment:
Is this module dependent upon aws-site-1, or is it just aws-base-1?

Reply:
I built this module without applying aws-vpc-site-1. So it appears that there isn't a dependency at this time.

Reply:
sorry, copy/paste error... Dependency only needs aws-base-1. I will correct.